docs/conf.py

```diff
@@ -263,12 +271,8 @@ def add_buttons(
 # -- Options for nbsphinx gallery------------------------------------------
 
 nbsphinx_thumbnails = {
-    'examples/gallery/nyc-taxi':
-    '_static/gallery/nyc-taxi-thumbnail.png',
-    'examples/gallery/masking-a-range':
-    '_static/gallery/masking-a-range-thumbnail.png',
-    'examples/gallery/rectangle-selection':
-    '_static/gallery/rectangle-selection-thumbnail.png',
-    'examples/gallery/scatter3d-with-threshold':
-    '_static/gallery/scatter3d-with-threshold-thumbnail.png',
+    'examples/gallery/nyc-taxi': '_static/gallery/nyc-taxi-thumbnail.png',
```

Comment:
line is too long

Reply:
Strange, is black not strict about the line length? Can you move the code around a little to make both tools happy?

Otherwise you would have to put a `noqa` comment.

Also, `flake8-bugbear` has a check that is a bit more lenient and only requires length 88-ish lines. We could use that instead of the builtin rule.